locksmith/migrations/20190205161953-create_transactions.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Transactions', {
+      transactionHash: {
+        type: Sequelize.STRING,
+        primaryKey: true
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
```

Comment:
and here too?

Reply:
This is also provide by the framework and will be the time of update